### PR TITLE
Remove --update-rte to ensure support of reproducible builds

### PR DESCRIPTION
**Pull request by Evagan2018**
- Remove --update-rte to ensure support reproducible builds
- Added GCC linker files located in RTE/Device directory
- Updated commandline description to generate the application on a local computer

### CODE REVIEW

File: Hello.cbuild-pack.yml
Comment (vovamarch) [inline Hello.cbuild-pack.yml:1]: Why is this file required?

**Review by vovamarch — CHANGES_REQUESTED** [review]

File: Hello.cbuild-pack.yml
Comment (Evagan2018) [reply Hello.cbuild-pack.yml:1]: According to **[cmsis-toolbox_build-overview_reproducible_builds](https://github.com/ReinhardKeil/cmsis-toolbox/blob/main/docs/build-overview.md#repository-version-control)** this file is essential for reproducible builds.

Reproducible builds are supported by the **[*.cbuild-pack.yml](https://github.com/ReinhardKeil/cmsis-toolbox/blob/main/docs/YML-CBuild-Format.md#file-structure-of-cbuild-packyml)** file that is created and read by the csolution project manager. This file:

- Records the exact list of the pack versions used during creation of an application.
- Ensures that pack versions do not change during development even when new packs are installed on the host computer, for example to develop other applications.

**Review by vovamarch — APPROVED** [review]
